CleverTapSDK/CleverTap.m

```diff
@@ -1707,6 +1717,9 @@ - (BOOL)inNotificationQueue {
 - (void)_showNotificationIfAvailable {
     if ([[self class] runningInsideAppExtension]) return;
     
+    //If inAppRenderingStatus is suspended do not continue showing inApps
+    if (self.inAppRenderingStatus == CleverTapInAppSuspend) return;
```

Comment:
Add logging as `InApp Notifications are set to be suspended, not showing the InApp Notification`